applications/FluidDynamicsApplication/python_scripts/navier_stokes_two_fluids_solver.py

```diff
@@ -252,10 +266,21 @@ def InitializeSolutionStep(self):
 
             # Update the DENSITY and DYNAMIC_VISCOSITY values according to the new level-set
             self._SetNodalProperties()
-
+          
             # Initialize the solver current step
             self._GetSolutionStrategy().InitializeSolutionStep()
 
+            # Accumulative Water volume error ratio due to level Set .Adding source term 
+            self.water_volume_after_transport=KratosCFD.FluidAuxiliaryUtilities.CalculateFluidNegativeVolume(self.GetComputingModelPart())
+
+            volume_error = (self.water_volume_after_transport -  self.system_volume)/ self.system_volume
+            self.volume_error_to_check_before_adding_source_term= self.water_volume_after_transport - self.system_volume
+            
+            # By setting VOLUME_ERROR to 0 the mass conservation source is set to 0. grad_v=0 
+            # self.GetComputingModelPart().ProcessInfo[KratosCFD.VOLUME_ERROR] = 0
+            self.GetComputingModelPart().ProcessInfo[KratosCFD.VOLUME_ERROR] = volume_error
```

Comment:
Rather than leaving this in here to deactivate the mass term, I'd add an option to the json settings. I'd do it inside the `formulation` section as we did with the `surface_tension` field.

Reply:
Done!